strings.Rmd

```diff
@@ -386,7 +386,7 @@ str_detect(c("grey", "gray"), "gr(?:e|a)y")
 1.  Describe, in words, what these expressions will match:
 
     1. `"(.)(.)\\2\\1"`
-    1. `(..)\1`
+    1. `"(..)\1"`
```

Comment:
Yes, of course. Those three variations make sense to help the reader understand the differences.


> On Jul 15, 2016, at 4:50 AM, Hadley Wickham <notifications@github.com> wrote:
> 
> In strings.Rmd <https://github.com/hadley/r4ds/pull/149#discussion_r70960193>:
> 
> > @@ -386,7 +386,7 @@ str_detect(c("grey", "gray"), "gr(?:e|a)y")
> >  1.  Describe, in words, what these expressions will match:
> >  
> >      1. `"(.)(.)\\2\\1"`
> > -    1. `(..)\1`
> > +    1. `"(..)\1"`
> I think the lack of quotes there was deliberate, to force you to think about the regular expression vs. the string the represents the regular expression
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/hadley/r4ds/pull/149/files/0bb63c42cc40a2177df212e491bfc6df704cad7a#r70960193>, or mute the thread <https://github.com/notifications/unsubscribe-auth/APMYab_krFz32IDtBOWshZWOremEnVzXks5qV3P2gaJpZM4JNBGt>.
>